lib/memo_wise.rb

```diff
@@ -425,25 +425,38 @@ def #{method_name}#{args_str}
       end
 
       # Override [Module#instance_method](https://ruby-doc.org/core-3.0.0/Module.html#method-i-instance_method)
-      # to proxy the original `UnboundMethod#parameters` results. This proxying
-      # helps code that uses Ruby reflection over method parameters
+      # to proxy the original `UnboundMethod#parameters` results. We want the
+      # parameters to reflect the original method in order to support callers
+      # who want to use Ruby reflection to process the method parameters,
+      # because the generated memoized method would have a generic set of
```

Comment:
Would you mind tweaking this to say something like `... because our overridden `initialize` method, in some cases the generated memoized methods, would have ...`? (Not sure if you're aware but avoiding generic params like `*args` in many cases is how @jemmaissroff and I got some big speedups recently.)

Reply:
Changed as requested!